units/systemd-networkd.service.in

```diff
@@ -25,9 +25,9 @@ RestartSec=0
 ExecStart=!!@rootlibexecdir@/systemd-networkd
 WatchdogSec=3min
 User=systemd-network
-DynamicUser=yes
```

Comment:
We have a systemd-network user right?

Reply:
I haven't seen were we define it. My guess right now is that systemd is creating that transparently for us? On the three machines I am running this it looks fine.

Reply:
Its in `modules/system/boot/systemd.nix`